CRM/Financial/BAO/FinancialItem.php

```diff
@@ -295,13 +295,28 @@ public static function checkContactPresent($contactIds, &$error) {
    * @return object CRM_Core_DAO
    */
   public static function getPreviousFinancialItem($entityId, $entityTable = 'civicrm_line_item') {
+    $liabilityAccountType = CRM_Core_OptionGroup::getValue('financial_account_type', 'Liability', 'name');
```

Comment:
Note that function is deprecated - if you use an IDE like php storm that leaps out at you.

![screenshot 2017-03-15 20 31 07](https://cloud.githubusercontent.com/assets/336308/23937825/5f90058c-09be-11e7-960b-bc76cb775d96.png)

I believe in more cases you can do things with 

CRM_Core_PseudoConstant::getLabel()
CRM_Core_PseudoConstant::getKey()
CRM_Core_Pseudoconstant::getName()